app/api-v1/controllers/Order/helpers.js

```diff
@@ -1,6 +1,32 @@
 const db = require('../../../db')
 const { NoTokenError, NothingToProcess } = require('../../../utils/errors')
 
+/*eslint-disable */
+const buildRecipeOutputs = (data, n) => new Array(n).fill({
+  roles: {
+    Owner: data.selfAddress,
+    Buyer: data.selfAddress,
+    Supplier: data.supplier,
+  },
+  metadata: { type: { type: 'LITERAL', value: 'RECIPE' } },
```

Comment:
we need to link these to the original input recipes via the `parent_index` rigth @jonmattgray